Router/invoice_query_state.js

```diff
@@ -1,4 +1,17 @@
 invoiceQueryState = {
+  LoadDefaultIfUrlNotCorrect: function () {
+    let timeRangeParams = FlowRouter.current().params.timeRange;
+    let sortOrderParams = FlowRouter.current().queryParams.sortOrder;
+    let sortedByParams = FlowRouter.current().queryParams.sortedBy;
+    if (!timeRangeParams || !sortOrderParams || !sortedByParams)
```

Comment:
The problem here is that you are setting the defaults if *any* of the params are not there.
It would be great if you just filled up the missing parts, so e.g: if the sort param is missing, just add a default one, but keep the other params.